spec/features/consumer/authentication_spec.rb

```diff
@@ -60,6 +60,13 @@
             page.should have_content "too short"
           end
 
+          scenario "Failing to sign up because email is already registered" do
+            fill_in "Email", with: user.email
+            fill_in "Choose a password", with: "foobarino"
+            click_signup_button
+            page.should have_content "There's already an account for this email."
```

Comment:
We should start using the "new" Rspec syntax:
`expect(page).to have_content "There's already an account for this email."`